ui/src/router/index.js

```diff
@@ -186,6 +186,9 @@ router.beforeEach((to, from, next) => {
   const hasLoggedIn = store.getters["user/hasLoggedIn"];
   const accountType = store.getters["user/accountType"];
   const hasBeenActivated = store.getters["user/hasBeenActivated"];
+  const hasAcceptedTnC = store.getters["user/hasAcceptedTnC"];
+  const hasAcceptedPrivacyPolicy = store.getters["user/hasAcceptedPrivacyPolicy"];
+  const hasAcceptedBoth = (hasAcceptedTnC && hasAcceptedPrivacyPolicy);
```

Comment:
Very minor remark; I think its better to rename this to hasAcceptedTncAndPp, because if I dont read the whole code, I wont know what both refer to XD otherwise ok

Reply:
I trust you that you have tested these, right?

Reply:
yes, i have tested it. it works fine. if user has accepted both TnC and PP, they can access the web app as usual. if not, they will be stuck on the user main page with the TnC and PP buttons; they can't access other pages by typing the url manually.

Reply:
Alright, thank you for the confirmation!

Reply:
Accepted, but I cant add my approval somehow XD